ood-portal-generator/spec/ood_portal_generator_view_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'spec_helper'
+require File.expand_path '../../lib/ood_portal_generator', __FILE__
+
+describe OodPortalGenerator::View do
+  describe 'dodes config opts match with example opts?' do
+    it 'returns true if opts match' do
+      config_opts = OodPortalGenerator::View.new
+        .instance_variables.map(&:to_s).map { |opt| opt.delete("@") }
+      example_config_opts = File.read(
+          './share/ood_portal_example.yml').scan(/#[\w_]+:/)
```

Comment:
Instead of `.scan(/#[\w_]+:/)` use `.scan(/#([\w_]+):/).flatten`

```
irb(main):038:0> x
=> "\n#one: 1\n#two: 2\n"
irb(main):039:0> puts x

#one: 1
#two: 2
=> nil
irb(main):040:0> x.scan(/#([\w_]+):/).flatten
=> ["one", "two"]
irb(main):041:0>
```